news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest Spotlight season released very recently, we are underway reworking osu! Spotlights from ground up, starting with a brand new team and adding a whole new league system.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a whole new twist.
```

Comment:
> by giving the curation and participation in osu! Spotlights a whole new twist.

this sounds like you're trying to introduce a surprise, but you've already mentioned the changes in the previous paragraph. i think it would be better to get to the point faster since your reader is already expecting to hear about new team and leagues

Reply:
Well, they just need to read onwards